tests/internal/torch/common_utils.py

```diff
@@ -132,7 +132,11 @@
 
 import torch
 import torch.cuda
-from torch._six import string_classes
+
```

Comment:
I wonder does torch remove this `torch._six` in higher versions? do you know which version?
It's better for us to remove these dependencies on private modules.

Reply:
@wangraying torch._six has been removed from PyTorch 2.0.0.